src/patches/homePatch.tsx

```diff
@@ -75,21 +75,13 @@ export const addHomePatch = (mounting = false, square = false, matchFeatured = f
 
   patch = routerHook.addPatch('/library/home', (props) => {
     afterPatch(props.children, 'type', (_: Record<string, unknown>[], ret?: any) => {
-      let cache2: any = null;
       wrapReactType(ret);
       afterPatch(ret.type, 'type', (_: Record<string, unknown>[], ret2?: any) => {
-        if (cache2) {
-          ret2 = cache2;
-          return ret2;
-        }
 
         let cache3: any = null;
         const recents = findInReactTree(ret2, (x) => x?.props && ('autoFocus' in x.props) && ('showBackground' in x.props));
 
-        wrapReactType(recents);
         afterPatch(recents.type, 'type', (_: Record<string, unknown>[], ret3?: any) => {
```

Comment:
`wrapReactType()` is here because `recents.type` is a memo, if memos don't get wrapped plugins don't unpatch properly during the usual events (reload, uninstall, etc) and would require the user to restart Steam to fix things. For the SGDB plugin specifically, I have it so that editing some of the settings unpatches then repatches the home, so this solution doesn't really work unfortunately.

Reply:
Oh, thanks for the info, I guess it's bad practice not to use wrapReactType then, I'll see if there's another way to fix this issue.

Reply:
question: i’m not versed in IT at all and was wondering what the fix was for this issue? how do i go about changing the code?